modules/farm_fd2/src/entrypoints/transplanting/transplanting.submitReset.e2e.cy.js

```diff
@@ -78,20 +78,47 @@ describe('Transplanting: Submit/Reset Buttons component', () => {
     cy.get('[data-cy="date-input"]').clear();
     cy.get('[data-cy="submit-button"]').click();
     cy.get('[data-cy="submit-button"]').should('be.disabled');
+    cy.get('[data-cy="date-input"]').type('2024-05-30');
+    cy.get('[data-cy="date-input"]').blur();
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
   });
 
   it('Invalid crop disables submit', () => {
     populateForm({ skipCrop: true });
     cy.get('[data-cy="submit-button"]').should('be.enabled');
     cy.get('[data-cy="submit-button"]').click();
     cy.get('[data-cy="submit-button"]').should('be.disabled');
+    cy.get('[data-cy="transplanting-picklist"]')
+      .find('[data-cy="selector-input"]')
+      .select('BROCCOLI');
+    cy.get('[data-cy="transplanting-picklist"]');
+    cy.get(`[data-cy="picklist-quantity-4"]`).select(2);
+    cy.get(`[data-cy="picklist-quantity-4"]`);
+    cy.get('[data-cy="submit-button"]').should('be.enabled');
   });
 
   it('Invalid location disables submit', () => {
     populateForm({ skipLocation: true });
     cy.get('[data-cy="submit-button"]').should('be.enabled');
     cy.get('[data-cy="submit-button"]').click();
     cy.get('[data-cy="submit-button"]').should('be.disabled');
+    cy.get('[data-cy="transplanting-location"]')
+      .find('[data-cy="selector-input"]')
+      .select('ALF');
+    cy.get('[data-cy="transplanting-location"]')
+      .find('[data-cy="picker-options"]')
+      .find('input')
+      .eq(0)
+      .click();
+    cy.get('[data-cy="transplanting-location"]')
+      .find('[data-cy="picker-options"]')
+      .find('input')
+      .eq(3)
+      .click();
+    cy.get('[data-cy="transplanting-location"]').find(
```

Comment:
This statement doesn't seem to do anything but find an element.  Should it do something with the element? Or is it unnecessary?